apps/src/pythonlab/pyodideWorkerManager.ts

```diff
@@ -61,8 +62,8 @@ const getMessageHandlers = (
     };
   } else {
     return {
-      writeConsoleMessage: () => {},
-      writePartialLine: () => {},
+      writeConsoleMessage: (message: string) => console.log(message),
```

Comment:
I see the case when pyodide is loading is below. Is this the case when pyodide is restarted? Just making sure I'm following this update!

Reply:
This is the case of before the user has clicked 'run', when we don't have a console manager set yet (we only set it on first run). 
`isPyodideLoading` will be set both on initial load and on restart, as it's triggered by the loading message.